chapters/annotations.tex

```diff
@@ -883,7 +884,7 @@ \subsection{Component Instance}\label{component-instance}
 For connectors, the icon layer is used to represent a connector when it is shown in the icon layer of the enclosing model.
 The diagram layer of the connector is used to represent it when shown in the diagram layer of the enclosing model.
 Protected connectors are only shown in the diagram layer.
-Public connectors are shown in both the diagram layer and the icon layer.
+Public connectors are shown in both the diagram layer and the icon layer, the sub-connectors in a hierarchical connector should only be shown when they can be connected to.
```

Comment:
Should be two sentences:
```suggestion
Public connectors are shown in both the diagram layer and the icon layer.
Sub-connectors in a hierarchical connector should only be shown when they can be connected to.
```

For a more readable text, the two sentences about protected and public could be combined into a single one:
> Both public and protected connectors are shown in the diagram layer, while only public connectors are shown in the icon.